charts/edgenode-dashboards/deployments/edgenode-dashboards/grafana-dashboards/edgenode-agent-logs.json

```diff
@@ -330,36 +330,16 @@
           },
           {
             "selected": false,
-            "text": "INBM",
-            "value": "INBM"
-          },
-          {
-            "selected": false,
-            "text": "INBM Dispatcher",
-            "value": "INBM_Dispatcher"
-          },
-          {
-            "selected": false,
-            "text": "INBM Configuration",
-            "value": "INBM_Configuration"
-          },
-          {
-            "selected": false,
-            "text": "INBM Diagnostic",
-            "value": "INBM_Diagnostic"
-          },
-          {
-            "selected": false,
-            "text": "INBM Telemetry",
-            "value": "INBM_Telemetry"
+            "text": "INBD",
+            "value": "INBD"
           },
           {
             "selected": false,
             "text": "INBM Update Status",
```

Comment:
Will we still be sending logs with this tag or should this be removed as well?

Reply:
update status log file is still there. so keeping it